src/Language.yml

```diff
@@ -75,41 +75,41 @@ English:
       This_Player_Is_Already_In_A_Clan: "&cThis name is already in a clan!"
-      Player_Has_Already_A_Invitation: "&cYou already invited this name!"
+      Player_Has_Already_A_Invitation: "&cYou already invited this player!"
       No_Clan: "&cYou are not in a clan!"
       Clan_Not_Exists: "&cThis clan does not exist!"
       Leaders_Can_Not_Leave_Clans: "&cYou can not leave your clan while you are the leader."
       Clan_Not_Deleted: "&7Your clan was &cnot &7deleted!"
       Clan_Not_Leaved: "&7You did &cnot &7leave your clan!"
       Player_Not_Kicked: "&7The player &b%player% &7was &cnot &7kicked!"
-      Player_Is_Offline: "&cThis name is not online!"
-      Player_Not_Exists: "&cThis name does not exist!"
+      Player_Is_Offline: "&cThis player is not online!"
+      Player_Not_Exists: "&cThis player does not exist!"
       No_Invite: "&cYou are not invited or your invite is expired!"
       No_Alliance_Invite: "&cYou are not invited to an alliance or your invite is expired!"
       Self_Invite: "&cYou can not invite yourself!"
       Self_Kick: "&cYou can not kick yourself!"
-      Other_Clan: "&cThis name is not in your clan!"
+      Other_Clan: "&cThis player is not in your clan!"
       Leaders_Can_Not_Be_Kicked: "&cLeaders can not be kicked out of the clan!"
       Chat_Is_Not_Enabled: "&cYour clan chat is not enabled!"
       Leader_Is_Not_Online: "&cThe leader of the other clan is not online."
-      Alliance_Between_Clans: "&cYou are in an alliance with this name."
+      Alliance_Between_Clans: "&cYou are in an alliance with this clan."
       Already_In_Alliance: "&cYour clan is already in an alliance with the clan &b%clan%&c!"
-      Same_Clan: "&cThis name is in your clan!"
+      Same_Clan: "&cThis player is in your clan!"
       Own_Clan: "&cThis is your clan!"
       No_Alliance: "&cThere is no alliance between the clan &b%clan% &cand your clan!"
       Already_Leader: "&cYou are already the leader!"
-      Already_Trusted: "&cThis name is already a %rank_color%trusted member&c!"
-      Already_Member: "&cThis name is already a %rank_color%member&c!"
+      Already_Trusted: "&cThis player is already a %rank_color%trusted member&c!"
+      Already_Member: "&cThis player is already a %rank_color%member&c!"
       Demote_Wrong_Command: "&cYou can not use this command, use the 'Leader' command!"
       Unknown_Page: "&cThis page is not available!"
       Clan_Name_To_Long: "&cYour clan name is to long, you can only use %num_of_chars% characters!"
       Clan_Max_Alliances: "&cYour clan already reached the maximum number of alliances."
       Clan_Already_Full: "&cYour clan already reached the maximum amount of members."
       This_Clan_Is_Full: "&cThis clan already reached the maximum amount of members."
       No_Clan_Base: "&cYour clan has no base."
-      Clan_Not_Neutralized: "&7You did &cnot &7neutralized this alliance!"
+      Clan_Not_Neutralized: "&7You did &cnot &7neutralize this alliance!"
```

Comment:
Maybe disbanded is better?

"&7You have not disband this alliance!"

Or mabye break?
"&7You have not broken this alliance"